client/blocks/plugins-update-manager/schedule-create.tsx

```diff
@@ -60,39 +63,50 @@ export const ScheduleCreate = ( props: Props ) => {
 	};
 
 	return (
-		<Card className="plugins-update-manager">
-			<CardHeader size="extraSmall">
-				<div className="ch-placeholder">
-					{ onNavBack && (
-						<Button icon={ arrowLeft } onClick={ onNavBack }>
-							Back
-						</Button>
+		<>
+			{ ! hasUserManagedPlugins && (
+				<Banner
+					title="No updatable plugins found"
+					description={ `You don't have any plugins that can be updated. Please head over to <a href="https://wordpress.com/plugins/${ siteSlug }">Plugins</a> to install some plugins.` }
+					onClick={ () => {
+						page.redirect( `/plugins/${ siteSlug }` );
+					} }
+				/>
+			) }
+			<Card className="plugins-update-manager">
+				<CardHeader size="extraSmall">
+					<div className="ch-placeholder">
+						{ onNavBack && (
+							<Button icon={ arrowLeft } onClick={ onNavBack }>
+								Back
+							</Button>
+						) }
+					</div>
+					<Text>New Schedule</Text>
+					<div className="ch-placeholder"></div>
+				</CardHeader>
+				<CardBody>
+					<ScheduleForm onSyncSuccess={ onSyncSuccess } onSyncError={ setSyncError } />
+				</CardBody>
+				<CardFooter>
+					<Button
+						form="schedule"
+						type="submit"
+						variant={ canCreateSchedules ? 'primary' : 'secondary' }
+						disabled={ ! canCreateSchedules }
```

Comment:
Should we disable the create button if there are no plugins to update?
(Not a leading question, genuinely don't know)

Reply:
I think we should.